doc/cephfs/eviction.rst

```diff
@@ -104,7 +104,7 @@ the blocklist:
 
 ::
 
-    $ ceph osd blocklist ls
+    $ ceph osd blacklist ls
```

Comment:
the `osd blacklist ls` command has been deprecated, and will be removed in future. see

https://github.com/ceph/ceph/blob/d7862cb6070094e611a031e5c8194235e2c503e6/src/mon/MonCommands.h#L1010-L1020

it is replaced by the `osd blocklist ls` command.

https://github.com/ceph/ceph/blob/d7862cb6070094e611a031e5c8194235e2c503e6/src/mon/MonCommands.h#L1001-L1008